saleor/product/utils.py

```diff
@@ -109,26 +109,31 @@ def products_for_cart(user):
     return products
 
 
-def product_json_ld(product, availability=None, attributes=None):
+def product_json_ld(product, attributes=None):
     # type: (saleor.product.models.Product, saleor.product.utils.ProductAvailability, dict) -> dict  # noqa
     """Generates JSON-LD data for product"""
     data = {'@context': 'http://schema.org/',
             '@type': 'Product',
             'name': smart_text(product),
-            'image': smart_text(product.get_first_image()),
+            'image': [
+                product_image.image.url
+                for product_image in product.images.all()],
```

Comment:
We could list images as `ImageObject`, providing dimensions and alt text

Reply:
We could but Google will only ever use the alt text (for image search) and it defaults to the product name.

Reply:
Right, this should be easily fetched by the crawler, let's leave it as it is.